cves/kernel/CVE-2017-18549.yml

```diff
@@ -55,7 +55,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description:  |
+  RAM is (virtually) segregated into two parts, the Kernelspace (where privileged kernel programs and drivers run) 
+  and the Userspace (where regular user software runs). In C, an uninitiated variable points to the address allocated in the Kernelspace, 
+  and the variable holds whatever was in there from before (which could be sensitive information). 
+  In this vulnerability this uninitialized variable's data is sent from the Kernelspace into the Userspace.
```

Comment:
Great explanation! Keep going - why is that a problem?

Reply:
added further explanation as to how this can be a problem.